inc/page.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Each sitemaps has total posts / 50,000 pages.
```

Comment:
This is a good idea for packing more URLs into a single page but is probably unnecessary for a first run. Let's start simple by mapping each sitemap page URL to a single bucket of 2000 posts.